README.md

```diff
@@ -1,74 +1,96 @@
 [![PyPI version](https://badge.fury.io/py/dsp-tools.svg)](https://badge.fury.io/py/dsp-tools)
 
 # DSP-TOOLS - DaSCH Service Platform Tools
-
-dsp-tools is a command line tool that helps you interact with the DaSCH Service Platform API.
-
+dsp-tools is a command line tool that helps you interacting with the DaSCH Service Platform API.  
 Go to [Full Documentation](https://dasch-swiss.github.io/dsp-tools/)
 
 ## Information for developers
+There is a `Makefile` for all the following tasks (and more). Type `make` to print the available targets. 
+
+For a quick start, use: 
+```
+pip install pipenv
+pipenv install --dev
+make run
+```
```

Comment:
`make run` executes `pip3 install -e .`, instead of `pipenv install -e .` Is that good or bad?

Reply:
then it's not guaranteed to run in the virtual environment. I suggest changing it to `pipenv run make run`

Reply:
I would even consider to use `pipenv install -e` in the Makefile for `make run`. Is there anything that speaks against it?

Reply:
> I would even consider to use `pipenv install -e` in the Makefile for `make run`. Is there anything that speaks against it?

Not everyone has pipenv installed.